episodes/narrative.md

```diff
@@ -45,6 +45,27 @@ You can enable learners to make connections between what they learn in your less
 own work by creating a narrative that resembles a situation the learners might
 encounter there.
 
+Here are some example narratives from lessons:
+
+* [Software Carpentry Git lesson](https://swcarpentry.github.io/git-novice/index.html) uses the story of 
+  Wolfman and Dracula who have been hired by Universal Missions (a space services spinoff from Euphoric State University) 
+  to investigate if it is possible to send their next planetary lander to Mars. 
+  They want to be able to work on the plans at the same time, but they have run into problems doing this in the past. 
+  They now need a system that will enable them to work collaboratibely and in parallel and track each change.
+* [Building Websites With Jekyll and GitHub lesson](https://carpentries-incubator.github.io/jekyll-pages-novice/filters/index.html) 
+  uses the narrative of creating a website for a research project to teach authoring webpages with Markdown and compiling them into a website served on GitHub.
+* [FAIR Research Software lesson](https://carpentries-incubator.github.io/fair-research-software/) uses the narrative of a poorly designed software project 
+  (which analyses [NASA’s open data on spacewalks](https://data.nasa.gov/Raw-Data/Extra-vehicular-Activity-EVA-US-and-Russia/9kcy-zwvn/data_preview) undertaken by astronauts from 1965 to 2013) that over the course of the lesson gets improved in terms of 
+  its accessibility, readability, interoperability and reusability to become more FAIR
+* This lesson uses [the recipe on how to bake sourdough cinnamon rolls](https://github.com/tobyhodges/cuddly-waffle) to 
+  teach how to use The Carpentries Workbench lesson template as we will see later on.
```

Comment:
```suggestion
```

I would leave this one off the list, as 1. I may delete that repo soon, and 2. I don't think the recipe example os a particularly good fit as an example for a lesson -- the idea of a 'narrative' doesn't really hold. I will try to think of an alternative.